.gitignore

```
@@ -127,4 +127,8 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+<<<<<<< HEAD
 .idea/
+=======
+.idea/
+>>>>>>> 64328121261725cfc20e9ee62050e9a38287013c
```

here is a conflict that should be resolved like:
```suggestion
.idea/
```